src/applications/personalization/appointments/actions/index.js

```diff
@@ -17,6 +20,34 @@ import MOCK_FACILITIES from '~/applications/personalization/dashboard-2/utils/mo
 import MOCK_VA_APPOINTMENTS from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_VA_APPOINTMENTS';
 import MOCK_CC_APPOINTMENTS from '~/applications/personalization/dashboard-2/utils/mocks/appointments/MOCK_CC_APPOINTMENTS';
 
+function isVideoVisit(appt) {
+  return !!appt.vvsAppointments?.length;
+}
+
+/**
+ * Finds the datetime of the appointment depending on the appointment type
+ * and returns it as a moment object
+ *
+ * @param {string} type 'cc' or 'va'
+ * @param {Object} appt VAR appointment object
+ * @returns {Object} Returns appointment datetime as moment object
+ */
+function getLocalAppointmentDate(type, appt) {
```

Comment:
this function is very heavily inspired by [a VAOS helper](https://github.com/department-of-veterans-affairs/vets-website/blob/3a690365264b41a566d3b99e923dc1123c9c8199/src/applications/vaos/services/appointment/transformers.js#L145) that does the same thing.